src/mds/objects.py

```diff
@@ -119,6 +122,68 @@ async def create_object(
     return JSONResponse(response, HTTP_201_CREATED)
 
 
+@mod.get("/objects/{guid:path}")
+async def get_object(
+    guid, request: Request,
+):
+    """
+    Get the metadata associated with the provided indexd GUID or alias.
+    If the GUID or alias does not exist in indexd, query the metadata
```

Comment:
i would reword this in a way that says it queries both either way. not necessarilly only query mds db if it doesn't exist in indexd